tests/ConnectionTest.php

```diff
@@ -63,6 +63,15 @@ public function testPing()
         $connection->ping();
     }
 
+    public function testQuota()
+    {
+        if (false !== \getenv('IMAP_QUOTAROOT_SUPPORTED')) {
```

Comment:
1. Please swap the logic here: if getenv is false, mark test skipped.
1. Could you please try to install the quota plugin in travis Dovecot installation? So we can prove your code works as expected:
    1. https://wiki2.dovecot.org/Quota/Configuration
    1. https://github.com/ddeboer/imap/blob/master/.travis/dovecot_install.sh

Reply:
I updated the test case logic and enabled quota plugin in travis dovecot installation. Seems to work fine. Please review again. ;)